libs/blockscout-db/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-db"
-version = "5.0.0"
+version = "6.0.0"
```

Comment:
version of blockscout-db should be the same as version of blockscout

Reply:
https://github.com/blockscout/blockscout-rs/pull/1041#discussion_r1751451240